R/data.table.R

```diff
@@ -1739,22 +1739,48 @@ replace_dot_alias = function(e) {
         GForce = FALSE
       } else {
         # Apply GForce
+        noCall_noVars = function(expr) (!is.call(expr) || length(all.vars(expr, max.names=1L))==0) && !dotN(expr)
```

Comment:
```suggestion
        noCall_noVars = function(expr) (!is.call(expr) || length(all.vars(expr, max.names=1L)) == 0L) && !dotN(expr)
```

Reply:
may as well set `all.vars(unique=FALSE)` as well?

Reply:
also: a comment on this function is warranted, ideally some examples of include/exclude expressions you have in mind. Function body itself is quite dense.